### PR TITLE
Use official MP-SPDZ repo

### PR DESCRIPTION
**Pull request by sbckr**
Signed-off-by: Sebastian Becker <sebastian.becker@de.bosch.com>

### CODE REVIEW

File: Dockerfile.spdz [hunk 1]
```diff
@@ -12,8 +12,8 @@ FROM ubuntu@sha256:10cbddb6cf8568f56584ccb6c866203e68ab8e621bb87038e254f6f27f955
 
 ARG VERSION
 ARG ARTIFACT_NAME="spdz"
-ARG SPDZ_VERSION="5350e66"
-ARG ARTIFACT_TAG="5350e66_no-offline"
+ARG SPDZ_VERSION="642d11f"
+ARG ARTIFACT_TAG="642d11f_no-offline"
```
Comment (strieflin) [inline Dockerfile.spdz:17]: No offline is the new normal. Would it make sense to remove the "no-offline" qualifier?

**Review by strieflin — CHANGES_REQUESTED** [review]

File: Dockerfile.spdz (hunk 1 above)
Comment (sbckr) [reply Dockerfile.spdz:17]: It definitely would, however we already have an image for the given version published. Reusing the tag would result in a conflict. I would therefore vote for keeping the qualifier and remove it once we update the spdz version itself

Comment (strieflin) [reply Dockerfile.spdz:17]: Can you maybe add that plan as a comment or create an issue to track it?

Comment (sbckr) [reply Dockerfile.spdz:17]: I have added a comment to follow up on this.

**Review by strieflin — APPROVED** [review]
LGTM

**Pull request merged**